src/main/java/com/epam/test_generator/services/EmailService.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.test_generator.services;
+
+import com.epam.test_generator.entities.Token;
+import com.epam.test_generator.entities.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.annotation.Resource;
+import javax.servlet.http.HttpServletRequest;
+
+@Transactional
+@PropertySource("classpath:email.messages.properties")
+@Component
+public class EmailService {
+    public final static Integer CONFIRMATION_TIME = 1440;
+    public final static Integer PASSWORD_RESET_TIME = 15;
+    @Autowired
+    private PasswordService passwordService;
+
+    @Autowired
+    private JavaMailSender emailSender;
+
+    @Autowired
+    private JavaMailSenderImpl javaMailSender;
+
+    @Autowired
+    private TokenService tokenService;
+
+    @Resource
+    private Environment environment;
+
+    public void sendRegistrationMessage(User user, HttpServletRequest request) {
+        Token userConformationToken = tokenService.createToken(user, CONFIRMATION_TIME);
+        String confirmUrl = passwordService.createConfirmUrl(request, userConformationToken);
+        String subject = environment.getProperty("subject.registration.message");
+        String text = environment.getProperty("registration.message");
+        text = String.format(text, "Maksim", "Stelmakh", "https://www.epam.com/",
```

Comment:
Let's use configuration files

Reply:
It will be complete after merging with master, the User Name and Surname will not be hard-code.